components/shared/searchFilters/SearchFiltersColumn.tsx

```diff
@@ -44,48 +45,38 @@ const SearchFiltersColumn = ({
     <>
       <div
         key={facet.name}
-        className={cn(["relative", !isLast && "min-w-32 flex-1", isLast && "flex-2"])}>
-        <h3 className="mb-2 text-typo-caption uppercase">{getFacetTranslation(facetFilter)}</h3>
+        className={cn(["relative ml-[-4px]", !isLast && "min-w-32 flex-1", isLast && "flex-2"])}>
+        <h3 className="mb-2 pl-2 text-typo-caption uppercase">
+          {getFacetTranslation(facetFilter)}
+        </h3>
         <div
           className={cn([
```

Comment:
remove the extra array

```suggestion
          className={cn(
```